README.md

```diff
@@ -1,106 +1,30 @@
 # Introduction
 
-This repository holds NVIDIA-maintained utilities to streamline mixed precision and distributed training in Pytorch.
-Some of the code here will be included in upstream Pytorch eventually.
+This repository holds NVIDIA-maintained utilities to streamline mixed precision and distributed training in PyTorch.
+Some of the code here will be included in upstream PyTorch eventually.
 The intent of Apex is to make up-to-date utilities available to users as quickly as possible.
 
 ## Full API Documentation: [https://nvidia.github.io/apex](https://nvidia.github.io/apex)
 
-## [GTC 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/GTC_2019) and [Pytorch DevCon 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/Pytorch_Devcon_2019) Slides
+## [GTC 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/GTC_2019) and [PyTorch DevCon 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/PyTorch_Devcon_2019) Slides
```

Comment:
Slides for GTC 2019 and PyTorch DevCon 2019 are available [here](https://github.com/mcarilli/mixed_precision_references/tree/b3b95a2265b392bc77a4de690dfddebb0ee48972)
```suggestion
## [GTC 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/GTC_2019) and [PyTorch DevCon 2019](https://github.com/mcarilli/mixed_precision_references/tree/master/PyTorch_Devcon_2019) Slides
```